<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametreMusiqueDialog</class>
 <widget class="QDialog" name="ParametreMusiqueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>337</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Paramètre  musique</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="sourceGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Source musical</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="interneRadio">
        <property name="text">
         <string>Interne</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cdromRadio">
        <property name="text">
         <string>CD-ROM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="externeRadio">
        <property name="text">
         <string>Externe</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="lancementGroupBox">
     <property name="title">
      <string>Lancement de la musique :</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horairesLancementLayout">
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="label1Layout">
          <item>
           <widget class="QLabel" name="OmatinLabel">
            <property name="text">
             <string>Avant l'ouverture le matin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FmidiLabel">
            <property name="text">
             <string>Après la fermeture le midi </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="OmidiLabel">
            <property name="text">
             <string>Avant l'ouverture le midi </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FsoirLabel">
            <property name="text">
             <string>Après la fermeture le soir </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QSpinBox" name="OmatinHeureSpin">
            <property name="maximum">
             <number>23</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="FmidiHeureSpin">
            <property name="maximum">
             <number>23</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="OmidiHeureSpin">
            <property name="maximum">
             <number>23</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="FsoirHeureSpin">
            <property name="maximum">
             <number>23</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="hOmatinLabel">
            <property name="text">
             <string>h</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hFmidiLabel">
            <property name="text">
             <string>h</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hOmidiLabel">
            <property name="text">
             <string>h</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hFsoirLabel">
            <property name="text">
             <string>h</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QSpinBox" name="OmatinMinuteSpin">
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="FmidiMinuteSpin">
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="OmidiMinuteSpin">
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="FsoirMinuteSpin">
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="randomCheckBox">
        <property name="text">
         <string>Lecture aléatoire de la musique</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Enregister</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeSaveButton">
       <property name="text">
        <string>Enregistrer et Fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ParametreMusiqueDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>ParametreMusiqueDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
